src/features/projects/components/maps/TimeTravel.tsx

```diff
@@ -68,14 +67,14 @@ export default function TimeTravel(): ReactElement {
       // A selector or reference to HTML element
       const container = '#comparison-container';
 
-      const _compare = new MapboxCompare(before, after, container, {
-        mousemove: false, // Optional. Set to true to enable swiping during cursor movement.
+      const compare = new MapboxCompare(before, after, container, {
```

Comment:
`compare` variable is not being used anywhere. Could you just declare the object without initializing the value to a variable.